release-engineering/role-handbooks/branch-manager.md

```diff
@@ -163,6 +164,18 @@ See "Cutting v1.15.0-alpha.2" under [References](#References) for an example Doc
 
 To leverage/contribute to our [release tools], Release Managers will need to fork and clone the [kubernetes/release] repo.
 
+Building and publishing releases require the latest revision of the release tools. The release tools can be compiled by running the following command from the [kubernetes/release] repository:
+
+```shell
+make release-tools
+```
+
+Additionally, the Kubernetes Release Toolbox (krel) requires the `cip-mm` tool to be present in the PATH. The `cip-mm` tool is used for promoting images/manifest. It can be found in the [kubernetes-sigs/k8s-container-image-promoter] repository, and obtained using `go get` such as:
+
+```shell
+go get sigs.k8s.io/k8s-container-image-promoter/cmd/cip-mm@master
+```
```

Comment:
I'm not sure is it fully-correct, but I've tested this command locally and it worked.

Reply:
I believe you will still need to `go install` (https://github.com/kubernetes/sig-release/blob/master/release-engineering/role-handbooks/release-image-promotion.md)

Reply:
Be aware that we're on our way to remove the dependency by using the image promoter as a library.

Reply:
> I believe you will still need to go install (https://github.com/kubernetes/sig-release/blob/master/release-engineering/role-handbooks/release-image-promotion.md)

I'll link to this document for instructions on how to install it.

> Be aware that we're on our way to remove the dependency by using the image promoter as a library.

@saschagrunert Should I add a note for that or just drop `cip-mm` as a requirement?

Reply:
I think you an leave it for now and add a HTML comment `TODO` that we have to remove it once the efforts in k/release got merged.

Reply:
@saschagrunert I've added a TODO.